### DISCUSSION TITLE
Uploading images and choosing what to share with other users

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [immich-app/immich#14180] [Q&A]
Hi, i just figured out how to add my image sharing to my domain for family to access, and noticed that i can share albums with other users i added on the immich container. Does that mean when they log in, they will not see any photos i've uploaded unless i create an album and share that specific album with those users?

Does the same regard for other users's photos? Can admin see those contents? Or is the privacy good enough to where those photos will be encrypted and admin not having access to those pictures? As i want to make it be a family shared image container where users can be safe knowing that what they upload, they alone have access to those medias. Or does admin account see it all unencrypted regardless?

**Comment by mmomjian** [comment]
Yes, each user will only see their own or shared photos, not photos of other users. 

The admin user can’t technically see anyone else’s photos, however the server owner can see all the pictures stored on the filesystem if they want to. There’s not really a way to avoid this.
